src/Registration/Metricsv4/RegisterTwoPointSets/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.10.2)
+
+project(RegisterTwoPointSets)
+
+find_package(ITK REQUIRED)
+include(${ITK_USE_FILE})
+
+
+add_executable(RegisterTwoPointSets Code.cxx Code.py)
```

Comment:
Here `Code.py` should be removed -- it is not part of the C++ executable.

Reply:
Removed. I'll note here that VS unfortunately does not include `Code.py` with the project when it is not included with the executable, which is more a matter of convenience than functionality. Also note that several other ITKExamples add Python files to the executable, perhaps unnecessarily.